BringYourOwnBoard.md

```diff
@@ -0,0 +1,157 @@
+> **DISCLAIMER:**  
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+# Introduction
+The goal of this document is to impart on the reader the necessary changes required to integrate the Device Update for IoT Hub agent with a board that isn't a RaspberryPi 4. These are not the total list of requirements to get your device ready to use an A/B Update System nor should it be considered a full guide on how to make your image Device Update Ready. 
+
+Instead we focus on the things that are completed by this repository with recommendations for further improvements and considerations the reader may make after reading. 
+
+You must work with your BSP provider to determine how to accomplish some of these tasks. 
+
+
+# Core Items to Prep an Image Build System
+
+To use Device Update for IoT Hub there are some requirements that your image build system must complete in order to work with the agent. 
+
+1. The prepared image MUST contain at least three partitions: the A partition, the B partition, and the ADU partition
+2. The prepared image MUST overload or modify the existing bootloader such that the system is able to load into either the A or B partition. The boot logic will be determined by your BSP. You must add the ability to set flags in order to use the correct partition after an update. You can see the one the proof-of-concept agent uses [here](./recipes-bsp/rpi-u-boot-scr/rpi-u-boot-scr.bbappend). Likely you will be able to append the image you're targeting's default bootloader but that will be on a case-by-case basis. 
```

Comment:
I found this a little difficult to parse: "Likely you will be able to append the image you're targeting's default bootloader". What about: "You will likely be able to append the default bootloader of the image you're targeting"?